modules/library/metadata/src/main/java/org/geotools/util/logging/CommonsLoggerFactory.java

```diff
@@ -61,7 +60,7 @@ public static synchronized CommonsLoggerFactory getInstance() throws NoClassDefF
      */
     protected Log getImplementation(final String name) {
         final Log log = LogFactory.getLog(name);
-        if (log instanceof Jdk14Logger) {
+        if (log.getClass().getName().endsWith("Jdk14Logger")) {
```

Comment:
There might exists classes in other packages than org.apache.commons.logging.impl with the same name Jdk14Logger. Because of that the check uses full qualified name